main.py

```diff
@@ -74,6 +74,9 @@ def setup_bot(bot: Bot) -> None:
     bot.load_extension("cogs.sudo")
     bot.load_extension("cogs.support")
     bot.load_extension("cogs.tools")
+    bot.load_extension("cogs.help")
+    # I still believe that it would be useful to, at least, have a dictionnary
+    # including all the extensions, for example in constants.py. That's quite a mess
```

Comment:
This comment doesn't really belong here, if you want to discuss this, please do so in the discord development channel, not in comments.
```suggestion
```